ext/include/opentelemetry/ext/http/client/curl/http_client_curl.h

```diff
@@ -228,7 +234,60 @@ class SessionManager : public http_client::SessionManager
     sessions_.erase(session_id);
   }
 
-  ~SessionManager() { curl_global_cleanup(); }
+  http_client::Result Get(const nostd::string_view &url,
+                          http_client::Headers &headers) noexcept override
+  {
+    HttpOperation curl_operation(http_client::Method::Get, url.data(), nullptr, RequestMode::Sync,
```

Comment:
Should `http_client::Method::Post` be used here?

Reply:
Yes, copy-paste error. Need to add test for POST sync request too. Thanks for pointing out :)

Reply:
This is fixed now.